BouncyCastle-JCA/src/Cipher.crysl

```diff
@@ -74,10 +74,12 @@ ORDER
 
 CONSTRAINTS
 	instanceOf[key, java.security.PublicKey] || instanceOf[key, java.security.PrivateKey] || encmode in {3, 4} => alg(transformation) in {"RSA"};
-	instanceOf[key, javax.crypto.SecretKey] => alg(transformation) in {"AES", "RIJNDAEL", "ElGamal", "ECIESwithAES-CBC", "DHIESwithAES-CBC", "Twofish", "Camellia", "Serpent", "Tnepres", 
-																		"Shacal2", "Shacal-2", "McEliece", "McEliecePointcheval", "McElieceKobaraImai", "McElieceFujisaki"};
-	noCallTo[Inits] => alg(transformation) in {"AES", "RSA", "RIJNDAEL", "ElGamal", "ECIESwithAES-CBC", "DHIESwithAES-CBC", "Twofish", "Camellia", "Serpent", "Tnepres", 
-																		"Shacal2", "Shacal-2", "McEliece", "McEliecePointcheval", "McElieceKobaraImai", "McElieceFujisaki"};
+	instanceOf[key, javax.crypto.SecretKey] => alg(transformation) in {"AES", "RIJNDAEL", "ElGamal", "ECIESwithAES-CBC", "DHIESwithAES-CBC",
```

Comment:
Is this already a preparation for CheckStyle introduction (characters per line)? Maybe check that it matches the coming rule.

Reply:
I initially thought of doing these changes for an easier readability, so we don't have to scroll horizontally in case of long lines. But good idea, we can a checkstyle for the rules. That would be better.